### PR TITLE
Add `activeworkspace` hyprctl command

### LINKED ISSUE #1442: activeworkspace IPC/hyprctl command
Yes I know this is quite easy to replicate, but it would be more concise and probably more performant if it was built in to the compositor!

### PR DESCRIPTION
**Pull request by yavko**
#### Describe your PR, what does it fix/add?
Adds `activeworkspace` hyprctl command, fixes #1442

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
no

#### Is it ready for merging, or does it need work?
Its ready

### CODE REVIEW

**Comment by yavko** [comment]
I can see there is some weird formatting, but I have no clue why, I ran `clang-format` on it

File: src/debug/HyprCtl.cpp [hunk 1]
```diff
@@ -14,6 +14,32 @@
 #include <sstream>
 #include <string>
 
+std::string activeWorkspaceRequest(HyprCtl::eHyprCtlOutputFormat format) {
```
Comment (vaxerski) [inline src/debug/HyprCtl.cpp:17]: if we're gonna do this then look at window requests and unify the data getting

File: hyprctl/main.cpp [hunk 2]
```diff
@@ -51,73 +52,73 @@ const std::string USAGE = R"#(usage: hyprctl [(opt)flags] [command] [(opt)args]
 )#";
 
 void              request(std::string arg, int minArgs = 0) {
-    const auto SERVERSOCKET = socket(AF_UNIX, SOCK_STREAM, 0);
+                 const auto SERVERSOCKET = socket(AF_UNIX, SOCK_STREAM, 0);
```
Comment (vaxerski) [inline hyprctl/main.cpp:55]: ?

**Review by vaxerski — CHANGES_REQUESTED** [review]

File: hyprctl/main.cpp (hunk 2 above)
Comment (yavko) [reply hyprctl/main.cpp:55]: Idk why, but `clang-format` is messing up stuff, I mentioned it in an above comment

File: src/debug/HyprCtl.cpp (hunk 1 above)
Comment (yavko) [reply src/debug/HyprCtl.cpp:17]: Alright, I'll make it like window requests when I get home

File: hyprctl/main.cpp (hunk 2 above)
Comment (vaxerski) [reply hyprctl/main.cpp]: ftfy

Comment (yavko) [reply hyprctl/main.cpp]: Thanks!

File: src/debug/HyprCtl.cpp (hunk 1 above)
Comment (yavko) [reply src/debug/HyprCtl.cpp]: resolved in [`a7c6d92` (#2202)](https://github.com/hyprwm/Hyprland/pull/2202/commits/a7c6d928613d941933d0f3a5ab7fe00417633516)

**Review by vaxerski — APPROVED** [review]
gangsta

**Pull request merged**